src/packages/perry/index.ts

```diff
@@ -65,19 +62,22 @@ export default class Perry {
     }
 
     setupListeners(this.options);
-
-    this.render();
   }
 
   public start = async () => {
-    if (this.options.clearOnReload) {
-      clearStore();
-    }
-
-    startListeners();
-
-    if (this.options.enableScreenRecording) {
-      await this.screenRecorder.start();
+    try {
+      if (this.options.clearOnReload) {
+        clearStore();
+      }
+
+      startListeners();
+
+      if (this.options.enableScreenRecording) {
+        await this.screenRecorder.start();
+      }
+    } catch (e) {
+      await this.stop();
```

Comment:
are you sure you want to die if the browser doesn't support the MediaDevices API?

Reply:
it will not die, but the error needs to be handled by the one trying to actually run this command

In react for example it doesn't breaks, but rather avoid's the next step code to run